### PR TITLE
Kdreval results setup

### PR DESCRIPTION
**Pull request by Kdreval**
cleanup of this repo + re-homed functions

### CODE REVIEW

**Comment by vladimirsouza** [comment]
Is there a reason for the current indent in DESCRIPTION?
```
...
Imports:
        ComplexHeatmap,
        DBI,
    RMariaDB,
        S4Vectors,
    circlize,
    config,
...
```

File: R/check_gambl_metadata.R
Comment (vladimirsouza) [inline R/check_gambl_metadata.R:57]: this function is missing as a dependency
```
check_gambl_metadata: no visible global function definition for
  'map_metadata_to_colours'
```

File: R/check_times.R
Comment (vladimirsouza) [inline R/check_times.R:35]: `R/check_times.R` prefixes some funtions from `lubridate` package, but this package is not included in DESCRIPTION.

File: R/calc_mutation_frequency_bin_region.R
Comment (vladimirsouza) [inline R/calc_mutation_frequency_bin_region.R:71]: This is probably a typo. It should be `:::` instead of `::` because `region_to_chunks` is an exported object from GAMBLR.helpers, instead of internal.

File: R/check_gambl_metadata.R
Comment (vladimirsouza) [reply R/check_gambl_metadata.R:57]: The function `map_metadata_to_colours` is going to be added to GAMBLR.helpers. After that, it can be imported here.

**Comment by Kdreval** [comment]
@vladimirsouza , if you fixed the things you picked up in the comments - can this be merged?

**Comment by vladimirsouza** [comment]
From my comments, the only thing that is still missing to address is
```
check_gambl_metadata: no visible global function definition for
  'map_metadata_to_colours'
```
I was waiting for `map_metadata_to_colours` to be added to GAMBLR.viz and then import it here. The migration to GAMBLR.viz was added in @mattssca's [PR](https://github.com/morinlab/GAMBLR.viz/pull/4/commits/e3d9b6f6164f84f6f525a1bed077857a6d5b3765), however this PR is on hold because of the changes in functions that call bundled data. 

Maybe we could remove `map_metadata_to_colours` migration from that PR and create a separate PR that only includes `map_metadata_to_colours` in GAMBLR.viz.

**Comment by mattssca** [comment]
That sounds like a good idea to me. Go ahead and create a new PR in GAMBLR.viz with this function.

**Comment by vladimirsouza** [comment]
All right. I can do that.

**Comment by mattssca** [comment]
Thanks! I can approve it once it's ready, just let me know when in case GitHub is slow with sending out the emails.

**Comment by vladimirsouza** [comment]
I added `GAMBLR.viz::` prefix to the internal call of `map_metadata_to_colours` in `check_gambl_metadata`. `check_gambl_metadata` was already working perfectly, I just added the prefix because this is what has been done for other functions to better track where functions come from.

**Comment by vladimirsouza** [comment]
For me, this PR could be approved. But, since there are also changes proposed by me, maybe would be better someone else check may changes as well. 

@Kdreval, if you can check my changes and approve them, please let me know and I will approve this PR.

**Comment by mattssca** [comment]
@vladimirsouza I have reviewed the changes you introduced in this PR and I think it's safe to merge if there are no outstanding issues that need attention. Thanks!

**Review by vladimirsouza — APPROVED** [review]

**Pull request merged**
